documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -3,16 +3,21 @@
 // assembly-kafka-connect.adoc
 
 [id='proc-deploying-kafkaconnector-{context}']
-= Deploying a `KafkaConnector` resource to Kafka Connect
+= Deploying a `KafkaConnector` resource
 
-This procedure describes how to deploy the example `KafkaConnector` to a Kafka Connect cluster.
+You can configure and deploy a `KafkaConnector` resource to a Kafka Connect cluster. 
+This creates a new connector instance with the specified configuration. 
 
-The example YAML will create a `FileStreamSourceConnector` to send each line of the license file to Kafka as a message in a topic named `my-topic`.
+This procedure creates a basic `FileStreamSourceConnector` that sends each line of the Kafka license file to a Kafka topic. 
+You can also create a `FileStreamSinkConnector`.
```

Comment:
Again, while we can mention it, we should point out that this basically has no real world use cases in a container environment, as least for the way Strimzi deployed Kafka Connect (or can we mount arbitrary volumes into the connect container now?)

Reply:
I don't know about the arbitrary volumes, but I added a new note to summarize this. 

NOTE
The `FileStreamSourceConnector` and `FileStreamSinkConnector` are not intended to be run in containers. 
In a production environment, you should prepare container images containing the Kafka Connect connectors that you want to use.